pycbc/inference/sampler/ultranest.py

```diff
@@ -22,16 +22,18 @@
 # =============================================================================
 #
 """
-This modules provides classes and functions for using the dynesty sampler
+This modules provides classes and functions for using the ultranest sampler
 packages for parameter estimation.
 """
 
 
 from __future__ import absolute_import
 
 import logging
+import numpy
 
 from pycbc.inference.io.ultranest import UltranestFile
+from pycbc.io.hdf import dump_state
```

Comment:
I see you're adding this import, but the checkpoint function still doesn't do anything. Are you planning on adding checkpointing in another PR?

Reply:
@cdcapano This was an obvious change. Even before checkpointing I need to figure out how to use it's MPI mode in our code. Otherwise, there won't be many problems fast enough to bother with.

Reply:
@cdcapano To be clear, I'm just using dump_state here to preserve the ultranest output in its original form if needed.